CPP11.md

```diff
@@ -933,6 +934,27 @@ C++11 introduces a memory model for C++, which means library support for threadi
 
 See the sections on: [std::thread](#stdthread)
 
+### Random functions
+According to CERT, the rand function doesn't have "good statistical properties" and can be predictable, which makes programs less secure (CERT Guideline MSC30-CPP), C++11 has a new and more secure library of tools for generating random numbers that can produce non-deterministic numbers for simulations and security cases, where predictability is not convenient. These new tools are found in the <random> header of the C++ standard library.
+An engine implements a random number generation algorithm that produces pseudo-random numbers. A distribution controls the range of values produced by an engine, the types of values (for example int, double, etc), and the statistical properties of the values.
+`std::default_random_engine` it´s the motor for default
+`std::uniform_int_distribution`evenly distributes pseudo-random integers across a specified range of values, the default range is 0 to the maximum value of an int on your platform
+```c++
+#include <iostream> // includes cout and endl
+#include <ctime> // includes time
+#include <random> // contains random number generation tools
+#include <assert.h> // include assert validation
+
+// use the generator motor of random numbers predetermined for 
+// produce pseudo-random values from 1 to 6, evenly distributed
+std::default_random_engine motor( static_cast<unsigned int>( std::time( 0 ) ) );
+std::uniform_int_distribution<unsigned int> intRandom( 1, 6 );
+
+const auto n = intRandom( motor ); // returns a random number between 1 to 6
```

Comment:
Let's remove some extra whitespace and code to help make the code snippet more succinct:
```cpp
std::default_random_engine motor(static_cast<unsigned int>(std::time(0)));
std::uniform_int_distribution<unsigned int> intRandom(1, 6);
const auto n = intRandom(motor); // returns a random number between 1 to 6
```

Reply:
I don't really to change anything this is from a collaboration by experts and authors, and the use of assert is really old, including the declaration of auto its just a bad practice.

Reply:
I'm a bit confused by your comments. Why does having this written by experts mean anything? None of the information is incorrect, I'm saying parts of it should be removed because the cheatsheet is supposed to be succinct: It's not meant to be a complete resource that goes into detail about everything. If users would like to learn about every feature in more depth, they should look into books or talks which dive into the subject more thoroughly.

You are welcomed not to make further changes, but I won't merge this as is in order to keep the material in the sheet as consistent as possible.